src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,238 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private User userWithCorrectFields;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        Storage.people.clear();
+
+        userWithCorrectFields = new User();
+        userWithCorrectFields.setLogin("thisLoginIsFit");
+        userWithCorrectFields.setPassword("thisPasswordIsFit");
+        userWithCorrectFields.setAge(21);
+    }
+
+    @Test
+    void register_nullUser_notOk() {
+        assertThrows(UserRegistrationException.class, () -> {
+            registrationService.register(null);
+        },
+                "It must be thrown UserRegistrationException for null user");
+    }
+
+    @Test
+    void register_nullLogin_notOk() {
+        userWithCorrectFields.setLogin(null);
+        assertThrows(UserRegistrationException.class, () -> {
+            registrationService.register(userWithCorrectFields);
+        },
+                "It must be thrown UserRegistrationException for null login");
+    }
+
+    @Test
+    void register_loginLessThanSixCharacters_notOk() {
+        userWithCorrectFields.setLogin("");
+        assertThrows(UserRegistrationException.class, () -> {
+            registrationService.register(userWithCorrectFields);
+        },
+                "It must be thrown UserRegistrationException for empty login");
+
+        userWithCorrectFields.setLogin("123");
+        assertThrows(UserRegistrationException.class, () -> {
+            registrationService.register(userWithCorrectFields);
+        },
+                "It must be thrown UserRegistrationException for login"
+                        + userWithCorrectFields.getLogin());
+
+        userWithCorrectFields.setLogin("12345");
+        assertThrows(UserRegistrationException.class, () -> {
+            registrationService.register(userWithCorrectFields);
+        },
+                "It must be thrown UserRegistrationException for login"
+                        + userWithCorrectFields.getLogin());
+    }
+
+    @Test
+    void register_loginMinLength_Ok() {
+        userWithCorrectFields.setLogin("login6");
```

Comment:
You should create User expected = new User(...., 
User actual = registrationService.register(expected);
assertEquals(expected, actual);


Valid cases should be checked by assertEquals, assertTrue etc